### DISCUSSION TITLE
How To Publish A Rest Endpoint With Http 1.0

### DISCUSSION THREAD

**Discussion by ilkyy** [uNetworking/uWebSockets.js#848] [Q&A] [answered]
Our load balancer checks applications sanity only with http 1.0 request. But uws server could not respond that. After a while request ended with nothing. How to handle http 1.0 request, any suggestions?

make a request with that curl:
`curl --http1.0 "http://localhost:9080/sanity"`

response:
`curl: (52) Empty reply from server`

this request responded successfully:
`curl --http1.1 "http://localhost:9080/sanity"`

Our uws code:
```
uWS.get("/sanity", handleSanity);

async function handleSanity(res, req) {
    res.cork(() => {
          res.writeHeader('Content-Type','application/json;charset=utf-8').writeStatus('200 OK').end('UP')
  })
}
```

**Comment by uNetworkingAB** [comment, marked as answer]
This project is HTTP 1.1 spec compliant and does not care for 30 year old deprecated-since-decades specs.
